client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -108,7 +108,7 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow, stepName } ) => {
 		'calypso_design_picker_image_optimization_202406'
 	);
 	const variantName = experimentAssignment?.variationName;
-	const oldHighResImageLoading = ! isLoadingExperiment && variantName === 'treatment';
+	const oldHighResImageLoading = true || ( ! isLoadingExperiment && variantName === 'treatment' );
```

Comment:
This is just for testing the treatment easier. Should get removed before merging.